index.html

```diff
@@ -59,13 +59,13 @@ <h1>Introducing Karma</h1>
     </div>
     </section>
     <section>
-    <div>
+    <article>
```

Comment:
I would prefer you to keep articles than mixing them with divs and the way you mixed them makes things furthermore complicated,

Reply:
I will use div for now, as I don't have enough time to re do my work now.

Reply:
ok cool, maybe later